src/components/ApplicationGroup/AppGroupAppFilter.components.tsx

```diff
@@ -92,7 +93,9 @@ export const Option = (props): JSX.Element => {
                 )
             }
         } else {
-            if (props.isFocused) {
+            // hide edit and delete icon for predefined filter
+            if (props.isFocused && props.data.value !== props.data.label) {
+                // for predefined filter, value and label are same (done in fe)
```

Comment:
Instead of using the same value and label for checking the predefined filter, we should try adding a type to the filer